src/components/button-minor/button-minor.component.tsx

```diff
@@ -1,14 +1,25 @@
-import React from "react";
+import React, { useContext } from "react";
 import StyledButtonMinor from "./button-minor.style";
 import { ButtonProps } from "../button";
+import { ButtonBarContext } from "../button-bar/button-bar.component";
 
 export const ButtonMinor = ({
-  buttonType = "secondary",
+  buttonType: buttonTypeProp = "secondary",
   size = "medium",
   ...rest
-}: ButtonProps) => (
-  <StyledButtonMinor size={size} buttonType={buttonType} {...rest} />
-);
+}: ButtonProps) => {
+  const { buttonType: buttonTypeContext } = useContext(ButtonBarContext);
```

Comment:
Good point @tomdavies73 the context needs expanding to support `size` and `iconPosition`, it looks like the context being consumed in `Button` is being superseded by the one consumed in `ButtonMinor` so you'll need to add that here

Reply:
This will explain the height discrepancies pointed out [here](https://github.com/Sage/carbon/pull/5988#pullrequestreview-1405962766), the width difference is fine as left and right border styling is being overridden in the ButtonBar styles